code/game/objects/items/devices/laserpointer.dm

```diff
@@ -146,9 +146,11 @@
 		var/mob/living/silicon/S = target
 		//20% chance to actually hit the sensors
```

Comment:
whats the current chance to actually fuck them

Reply:
If you're talking about the [1/2/3] stun?
(prob(effectchance * diode.rating)) * (prob(3 * diode.rating))
[20/40/60]% chance of hitting the sensors, and a further [3/6/9]% chance of doing the stun. Combined, that means it's a [0.6/2.4/5.4]% each time you shine the laser pointer to actually do the stun. It's really not very likely.

Reply:
yeah that sounds solid to me thanks